### PR TITLE
SAK-32572 Messages Tool: Allow all Member Roles from the Groups of wh…

### PR DESCRIPTION
**Pull request by manuelal**
…ich the current user is a member in the 'To' field

### CODE REVIEW

File: msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java [hunk 1]
```diff
@@ -1794,6 +1794,24 @@ private boolean isAllowToFieldMyGroupMembers(User user)
     else
       return false;
   }
+  
+  @Override 
+  public boolean isAllowToFieldMyGroupRoles() {
+	 LOG.debug("isAllowToFieldMyGroupRoles()");
```
Comment (ern) [inline msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java]: logging should be more descriptive and the method name is automatically output so no need to for it.

File: msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java [hunk 2]
```diff
@@ -1794,6 +1794,24 @@ private boolean isAllowToFieldMyGroupMembers(User user)
...
+	 return isAllowToFieldMyGroupRoles(userDirectoryService.getCurrentUser());
+	 }
+  
+  private boolean isAllowToFieldMyGroupRoles(User user)
+  {
+	  if (LOG.isDebugEnabled())
```
Comment (ern) [inline msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java]: isDebugEnabled is not needed.

File: msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java [hunk 3]
```diff
@@ -1794,6 +1794,24 @@ private boolean isAllowToFieldMyGroupMembers(User user)
...
+	  {
+		  LOG.debug("isAllowToFieldMyGroupRoles(User " + user + ")");
```
Comment (ern) [inline msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java]: Use log4j style parameter logging.

**Comment by manuelal** [comment]
Hi Earl,
Changes have been addressed.
Thanks.

**Pull request merged**